tableauserverclient/server/request_factory.py

```diff
@@ -209,6 +209,29 @@ def update_req(self, schedule_item):
                 single_interval_element.attrib[expression] = value
         return ET.tostring(xml_request)
 
+    def _add_to_req(self, id_, type_):
+        """
+        <task>
```

Comment:
Man we should have done this for all the request factories, it's really nice.

Reply:
yeah... hindsight...